paparazzi-gradle-plugin/src/test/projects/prepare-resources-task-caching/build.gradle

```diff
@@ -17,6 +17,9 @@ android {
   kotlinOptions {
     jvmTarget = libs.versions.javaTarget.get()
   }
+  dependencies {
+    implementation "androidx.core:core:1.10.0"
+  }
```

Comment:
decided to add the dep here instead of a separate test project.

Reply:
see `verify-update-aar-resources-change/build.gradle`